### PR TITLE
fix: unify all time into the time zone where NYC is located

### PR DESCRIPTION
**Pull request by MineRiaaa**

### CODE REVIEW

File: src/test/java/whu/edu/cs/transitnet/randomTest.java [hunk 1]
```diff
@@ -1,9 +1,25 @@
 package whu.edu.cs.transitnet;
 
+import org.junit.Test;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class randomTest {
```
Comment (xiaohaoxing) [inline src/test/java/whu/edu/cs/transitnet/randomTest.java:9]: Class name 是 UpperCamel

File: src/test/java/whu/edu/cs/transitnet/randomTest.java [hunk 2]
```diff
@@ -1,9 +1,25 @@
...
+        String startTime = "2023-04-05 13:44:00";
+        Date parse = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").parse(startTime);
+        Long time = parse.getTime();
+
+        Date d = new Date();
+        d.setTime(time);
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
 
+        String date = sdf.format(d);
+        System.out.println(date);
```
Comment (xiaohaoxing) [inline src/test/java/whu/edu/cs/transitnet/randomTest.java:22]: Test 的最后应该是一些 assert 来判断结果是否正确。

**Review by xiaohaoxing — APPROVED** [review]

File: src/test/java/whu/edu/cs/transitnet/randomTest.java (hunk 1 above)
Comment (MineRiaaa) [reply src/test/java/whu/edu/cs/transitnet/randomTest.java:9]: ok

File: src/test/java/whu/edu/cs/transitnet/randomTest.java (hunk 2 above)
Comment (MineRiaaa) [reply src/test/java/whu/edu/cs/transitnet/randomTest.java:22]: ok

**Pull request merged**